experimental/javascript.html

```diff
@@ -0,0 +1,161 @@
+<script>
+
```

Comment:
Can you add a comment on what the functions of this file (javascript.html) accomplish. I can understand that this is javascript responsible in manipulating the UI sidebar. But lets also add that here.

Reply:
Okay, I will add it in brief